### ISSUE TITLE
Historical quotation with adjusted prices

### ISSUE THREAD

**Issue by juvebianco** [mfinancecombr/site#1]
It would be extremely useful for those who like to calculate fundamentalist indicators (eg P/L) to have prices adjusted to avoid glaring distortions in the results of their calculations and consequently incurring erroneous decisions.

**Comment by marcelometal** [comment]
good point, but for now, you can use the endpoint below to get the P/L indicator:

https://mfinance.com.br/api/v1/stocks/indicators/COGN3
